com.ozmartians.VidCutter.json

```diff
@@ -0,0 +1,333 @@
+                    "url": "https://github.com/mstorsjo/fdk-aac/archive/v0.1.6.tar.gz",
+                    "sha256": "adbcd793e406e1b88b3c1c41382d49f8c27371485b823c0fdab69c9124fd2ce3"
+                }]
+            },
+            {
+                "name": "lame",
+                "config-opts": [
+                    "--enable-shared",
+                    "--disable-static",
+                    "--disable-frontend",
+                    "--enable-nasm"
+                ],
+                "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/lame/lame-3.100.tar.gz",
+                    "sha256": "ddfe36cab873794038ae2c1210557ad34857a4b6bdc515785d1da9e175b1da1e"
+                }]
+            }]
+        }]
+    },
+    {
+        "name": "mediainfo",
+        "cleanup": ["/include", "/lib/pkgconfig", "/share"],
+        "subdir": "Project/GNU/CLI",
+        "sources": [
+        {
+            "type": "archive",
+            "url": "https://mediaarea.net/download/source/mediainfo/18.03.1/mediainfo_18.03.1.tar.xz",
+            "sha256": "79664c356d4f09dc32105810aacf60923f6edcc6bec4836589962bc3375efcd6"
+        }],
+        "modules": [
+        {
+            "name": "libmms",
+            "config-opts": ["--disable-static"],
+            "sources": [
+            {
+                "type": "archive",
+                "url": "https://download.sourceforge.net/libmms/libmms-0.6.4.tar.gz",
+                "sha256": "3c05e05aebcbfcc044d9e8c2d4646cd8359be39a3f0ba8ce4e72a9094bee704f"
+            }]
+        },
+        {
+            "name": "libzen",
+            "config-opts": ["--enable-shared"],
+            "subdir": "Project/GNU/Library",
+            "sources": [
+            {
+                "type": "archive",
+                "url": "https://mediaarea.net/download/source/libzen/0.4.37/libzen_0.4.37.tar.xz",
+                "sha256": "401c34d93056a95f8392982f856d1371c5de6aec373d84dcbb356e1f473d0452"
+            }]
+        },
+        {
+            "name": "libmediainfo",
+            "config-opts": [
+                "--disable-static",
+                "--enable-shared",
+                "--disable-static",
+                "--with-libcurl",
+                "--with-libmms"
+            ],
+            "subdir": "Project/GNU/Library",
+            "sources": [
+            {
+                "type": "archive",
+                "url": "https://mediaarea.net/download/source/libmediainfo/18.03.1/libmediainfo_18.03.1.tar.xz",
+                "sha256": "0243365f53299997bc04dfa065de9b3f1688b6eeb450a7f57e740504cd626fa0"
+            }]
+        }]
+    },
+    {
+        "name": "vidcutter",
+        "buildsystem": "simple",
+        "build-commands": [
+            "pip3 install --prefix=${FLATPAK_DEST} ."
+        ],
+        "sources": [
+        {
+            "type": "git",
+            "branch": "flatpak",
```

Comment:
Needs a stable commit too.

Reply:
@TingPing thats actually the exact purpose for the new branch.. the flatpak branch will be updated after a few days after releases to master, so the usual unforeseen bugs can be dealt following general user feedback... if it needs to be a specific commit on the branch rather then HEAD then thats fine, i'll update the manifest now... 

cheers

Reply:
That doesn't change my comment. All Flatpaks hosted on Flathub need to be reproducible and that means pointing to a static commit.